cmd/parallel-manager.go

```diff
@@ -195,7 +195,10 @@ func (p *ParallelManager) enoughMemForUpload(uploadSize int64) bool {
 	}
 
 	smem := runtime.MemStats{}
-	runtime.GC()
```

Comment:
Maybe we don't need to run GC at all.. This is only to prevent mc crash due to the automatic parallel uploads

Reply:
@vadmeste I don't know all the logic here - that was also my initial solution, but since it wouldn't hurt on big files I just added the check.

Reply:
@vadmeste I will do what you prefer :)

Reply:
> @vadmeste I will do what you prefer :)

Let's keep what you did right now. I just remembered the context more, it is mainly for users who are using mc in VMs with low memory (2 or 4 GB). We had multiple issues before but now it is quiet. 50MB is a good trade off.